main.go

```diff
@@ -218,143 +214,51 @@ func run() (err error) {
 		log.Error().Err(err).Msg("Error initializing DB")
 		return
 	}
-	txnStore := repository.NewDBTxnStore(db, pc)
-	time.Sleep(5 * time.Second)
-	log.Info().Msg("Sleeping to allow DB to initialize tables")
-
-	tracer := chaintracer.NewSmartContractTracer(ctx, chaintracer.SmartContractTracerOptions{
-		ContractClient:      rc,
-		L1Client:            l1Client,
-		StartingBlockNumber: *startBlockNumber,
```

Comment:
We'll need a starting-block to make sure the service remains behind head for e2e testing.